src/layout/Tabbed.js

```diff
@@ -61,14 +62,32 @@
 
     Tabbed.prototype.enter = function (domNode, element) {
         HTMLWidget.prototype.enter.apply(this, arguments);
-        this._tabContainer = element.append("div");
         this._contentContainer = element.append("div");
+        this._contentContainer.attr("id", "content_container");
     };
 
     Tabbed.prototype.update = function (domNode, element) {
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;
 
+        // if (this._tabContainer) {this._tabContainer.remove();}
+        // this._tabContainer = element.insert("div", this.tabLocation() === "bottom" ? null : "div");
+        // this._tabContainer.attr("class", "on_" + this.tabLocation());
+
+        this._tabContainer = element.selectAll("#tab_contanier").data([this.tabLocation()], function(d) {return d;});
+        this._tabContainer
+            .enter()
+            .insert("div", function(d1) {
+                var x = d1 === "top" ? "#content_container" : null;
+                console.log(x, d1);
+                return x;
+            })
```

Comment:
You can't use tab_container ID in this way - mainly because you might have tabbed containers within tabbed containers and the selectAll will select All of them.  Look at the Grid example where it does things like:
```
var rows = this.contentDiv.selectAll(".cell_." + this._id).data(this.content(), function (d) { return d._id; });
```